capstone/src/main/webapp/data_viz.js

```diff
@@ -168,4 +153,132 @@ function mouseInToRegion(e) {
 function mouseOutOfRegion(e) {
   // Reset the hover state, returning the border to normal
   e.feature.setProperty('state', 'normal');
+}
+
+/* get County data from NYT in the form of a dictionary to put on the graph*/
+//date,county,state,fips,cases,deaths,confirmed_cases,confirmed_deaths,probable_cases,probable_deaths
+function getCounties(variable){
+  fetch('/getCountyData').then(response => response.text()).then((output) => {
+    let nyt_data = {};
+    let nyt_data_by_state = {};
+    let state_names = [];
+
+    let data_lines = JSON.parse(output);
+    for(let i = 0; i < data_lines.length; i++){
+      if(data_lines[i] !== null){
+        let parsed_data = data_lines[i].split(',');
```

Comment:
Not a big deal, but it might be better to serialize and deserialize the data as JSON instead of serializing and deserializing as a comma-separated list.